src/main/java/org/jspecify/nullness/NonNull.java

```diff
@@ -23,10 +23,11 @@
 import java.lang.annotation.Target;
 
 /**
- * Indicates that the annotated type usage is considered to exclude {@code null} as a value; rarely
- * needed within {@linkplain NullMarked null-marked} code.
+ * Indicates that the annotated <a href="http://github.com/jspecify/jspecify/wiki/type-usages">type
+ * usage</a> (commonly a variable type or return type) is considered to <i>exclude</i> {@code null}
```

Comment:
(We might have had this conversation already?)

It's certainly defensible to refer to fields and parameters as "variables" (e.g., [`VariableElement`](https://docs.oracle.com/en/java/javase/19/docs/api/java.compiler/javax/lang/model/element/VariableElement.html)). However, I think most people are likely to _initially_ read "variable" as referring to a local variable and in particular to the "top-level" type of that local variable. We of course explain elsewhere that that's _not_ where `@NonNull` can be used, but I still feel that we're getting off on the wrong foot.

I suggest changing "variable" to "parameter" or changing the whole parenthetical to something like "for example, a return type."

(Obviously, to the degree that you buy that, it applies to `@Nullable`, too.)

Reply:
I agree.

Reply:
Yeah great point, done x2